python/build.py

```diff
@@ -24,112 +24,79 @@
 section_names = section_names['section_names']
 
 print("section_names", section_names)
```

Comment:
Should the `all_path` stuff above go away too?

Reply:
Yes, good catch.